package.json

```diff
@@ -1,61 +1,63 @@
 {
-  "name": "@smg-automotive/example",
+  "name": "@smg-automotive/eslint-config",
   "version": "0.0.0-development",
   "description": "A boilerplate package setup",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "files": [
-    "dist/**/*"
-  ],
   "publishConfig": {
     "access": "public"
   },
   "scripts": {
     "version": "npm run build",
-    "build": "rimraf dist && rollup -c",
-    "lint": "eslint --ext ts,js,tsx,jsx,json .",
+    "build": "rimraf dist && tsc -p tsconfig.build.json --outDir dist",
+    "lint": "npm run build && eslint --ext ts,js,tsx,jsx,json ./src --ignore-path ./.eslintignore.internal",
     "format": "npm run lint -- --fix",
     "typecheck": "tsc --noEmit",
-    "test": "jest",
-    "test:debug": "node --inspect-brk --inspect=127.0.0.1:9229 ./node_modules/jest/bin/jest.js --runInBand"
+    "test": "npm run build && node dist/__tests__/test.js"
   },
   "author": "SMG Automotive",
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/smg-automotive/fe-example-pkg.git"
+    "url": "https://github.com/smg-automotive/eslint-config-pkg.git"
```

Comment:
is this supposed to be in `fe-eslint-config-pkg`? like.. is the `fe` a prefix we agreed upon? I don't care which way around, but in any case, please keep it consistent :)